### ISSUE TITLE
Seafile Community or Seafile Pro? Info screen...

### ISSUE THREAD

**Issue by StefanKrauth** [haiwen/seafile#1459]
Hi,
I upgraded to seafile pro. The info screen still displays "Community Edition". Now I'm uncertain if I did something wrong. The pro features seem to be active...

![bildschirmfoto 2015-11-24 um 22 26 25](https://cloud.githubusercontent.com/assets/663776/11381178/ceda6090-92fa-11e5-901b-9623195514c8.jpg)

Best Regards,
Stefan

**Comment by xiez** [comment]
Do you put seafile-license.txt file in install-path ? or is your pro version just a trial version with only 3 free user accounts ?

**Comment by StefanKrauth** [comment]
It's the trial as I will be the only user.

**Comment by StefanKrauth** [comment]
Oh interesting. I have created an empty seafile-license.txt and restarted seafile. It complained about the file not being valid. Then I deleted the file again and restarted seafile. It started up and is now showing "Professional Edition".

![sf](https://cloud.githubusercontent.com/assets/663776/11392667/7dd244c2-935b-11e5-8aec-e309f8ee18a5.png)

It shows Limit "--", but I can't create more than 3 users as expected.
